### PR TITLE
Add filtering to the source activities on edit Collection view.

### PR DESCRIPTION
**Pull request by idegtiarov**

### CODE REVIEW

File: bridge_adaptivity/module/static/module/js/module.js [hunk 1]
```diff
@@ -1,6 +1,22 @@
...
     $(function () {
         var defaultSourceItemTitle = "No Title";
+        var $filter = $("#filter"),
+            filterVal,
+            regexFilter;
+
+        $filter.change(function () {
+            filterVal = $("#filter").val();
+            if (filterVal) {
+                regexFilter = new RegExp(filterVal,"i");
```
Comment (alexbojko) [inline bridge_adaptivity/module/static/module/js/module.js]: add space after comma

File: bridge_adaptivity/module/static/module/js/module.js [hunk 2]
```diff
@@ -128,6 +144,11 @@
                 } else {
                     sourceButton.text(item["display_name"]);
                 }
+                if (regexFilter) {
+                    if (!regexFilter.exec(sourceButton.text())) {
+                        listItem.detach()
```
Comment (alexbojko) [inline bridge_adaptivity/module/static/module/js/module.js]: if you want to use `.detach()` method you should store somewhere value, turned by this function, and then, when you will want to append it somewhere, you will use this value. (something like cache or memo var).

In this case, using `.detach()` is not needed, because once you call `detach()` and not assign returned value to some variable you loose this items, and browser will not know what to insert in future. So, here you can use just `.remove()`, because logic of this app is to build DOM every time. 

Here's example from api.jquery.com about `detach()` function:

```
var p;
$( "button" ).click(function() {
  if ( p ) {
    p.appendTo( "body" );
    p = null;
  } else {
    p = $( "p" ).detach();
  }
});
```

Aws you can see, they store all `p` tags in global variable when they do `detach()` and from other side `if(p)`  they append them to body.

So, no magic here.

File: bridge_adaptivity/module/static/module/js/module.js (hunk 1 above)
Comment (idegtiarov) [reply bridge_adaptivity/module/static/module/js/module.js]: done

File: bridge_adaptivity/module/static/module/js/module.js (hunk 2 above)
Comment (idegtiarov) [reply bridge_adaptivity/module/static/module/js/module.js]: updated with the `remove` method
